app/controllers/ems_physical_infra_controller.rb

```diff
@@ -18,6 +18,21 @@ def self.table_name
     @table_name ||= "ems_physical_infra"
   end
 
+  def show_list
+    super
+    check_providers_credentials
+  end
+
+  # Check if there are providers with invalid credentials
+  # if it's true, adds an alert message to user
+  def check_providers_credentials
+    model.find_each do |provider|
+      if provider.authentication_status.casecmp("invalid").zero?
+        add_flash(_("The provider \"%{name}\" is with invalid credentials") % {:name => provider.name}, :error)
```

Comment:
`is with invalid credentials` should be `has invalid credentials`, for proper English.